src/wallet/wallet2.cpp

```diff
@@ -7579,6 +7579,40 @@ uint64_t wallet2::get_base_fee()
   return get_dynamic_base_fee_estimate();
 }
 //----------------------------------------------------------------------------------------------------
+uint64_t wallet2::get_base_fee(uint32_t priority)
+{
+  // clamp and map to 0..3 indices, mapping 0 (default, but should not end up here) to 0, and 1..4 to 0..3
+  if (priority == 0)
+    priority = 1;
+  else if (priority > 4)
+    priority = 4;
+  --priority;
```

Comment:
Think this general logic (starting from the comment) belongs inside `if (use_2021_scaling)`, otherwise this causes a discrepancy with current v17.3.0 clients until the hard fork (`get_fee_multiplier(priority)` gets called with a value of `priority` 1 less than it should be called with down in the else)